src/rt/sched/cown.h

```diff
@@ -842,7 +936,7 @@ namespace verona::rt
           //   return true;
           //
           // TODO: Investigate systematic testing coverage here.
```

Comment:
Is there an issue for this TODO?

Reply:
I wouldn't require every todo to have its own issue yet. Is there a reason why you want this TODO in particular to have an issue?

Reply:
In my experience, TODOs without issues never get addressed.

Reply:
And too many issues often get ignored. :D

At Arm, we had a script that would automatically create an issue for every build breakage in the CI. Builds were fixed, but after a few months, we had so many issues that we had no idea had been fixed or not that we just left them there.

Reply:
I don't see how that is relevant. We have issues to track work that needs to be done. If it needs to be done, there should be an issue.

Reply:
I see. I think my thought process comes from the way I use the TODO comment.

To me, TODO and FIXME are at least potential issues, not necessarily real issues. So, for example, I could add two TODOs in one file, while I only need to implement one and the other is "done". Or, I could add a FIXME and later realise it wasn't important.

More commenly, though, is adding TODOs for ideas that could be relevant, but the code gets refactored and all of those lose meaning and become irrelevant.

Tracking all of those changes with all issues in Github is not a trivial amount of work and doesn't scale well with the number of issues. We'd need TODOs with #number identifiers to make sure we track them, etc.

My general approach to bureaucracy is to do exactly as much as necessary, not more, not less. With a small team, a fast moving target, and a lot of code being produced and changed every month, I think that creating a tracking system for everything we're doing is overkill and will likely increase our overhead.

Reply:
I'm strongly against this. I think we should discuss with the larger team.